services/ifrs/src/ifrs/migration_logic/_utils.py

```
@@ -97,7 +102,7 @@ async def auto_finalize(self, coll_names: str | list[str], copy_indexes: bool):
 
             # copy indexes if needed (not implemented yet)
             if copy_indexes:
-                pass
+                raise NotImplementedError("Index copying is not yet implemented")
```

One last thing here:
If this raises, it would be better to move this check before the yield, so this fails early.
Else this always tries to do the migration first only to unstage all changes again.

---

fair enough. Done